hack/update-codegen-common.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generate mocks for testing with mockgen.
+function generate_mocks {
+  # Command mockgen does not automatically replace variable YEAR with current year
+  # like others do, e.g. client-gen.
+  current_year=$(date +"%Y")
+  sed -i "s/YEAR/${current_year}/g" hack/boilerplate/license_header.raw.txt
+  for target in "${MOCKGEN_TARGETS[@]}"; do
```

Comment:
Better to pass MOCKGEN_TARGETS as an argument of the func?

Reply:
Done

Reply:
Hi @jianjuns  I reset this change and using variable `MOCKGEN_TARGETS` directly without argument. The element in the array `MOCKGEN_TARGETS` has empty space which will be split as different parameters if we pass the array as an argument.

Reply:
We can pass the array elements to a func? https://stackoverflow.com/questions/16461656/how-to-pass-array-as-an-argument-to-a-function-in-bash

Reply:
It only works when the array's element has no empty space, if the element is like `pkg/agent/cniserver/ipam IPAMDriver testing`, it will be treated as three elements after we pass the array as an argument. I tried it with or without double quotes, it doesn't work as we expected.

Reply:
Ok. Guess we can change the delimiter from space to something like ";", but it is not a big deal and I am fine with the current way too.